src/main.c

```diff
@@ -364,7 +376,7 @@ int main(int argc, char **argv)
 
                 time_t now = time(NULL);
                 if (difftime(now, last_spool_run_time) >= spool_process_time) {
```

Comment:
I think ```delay``` should be part of the statement (when delay is in use), otherwise the loop will skip the spool processing if the time between the last pass and the time at of execution of the statement is less than ```spool_process_time``` (900 seconds by default).